.github/workflows/main.yml

```diff
@@ -92,20 +92,14 @@ jobs:
         uses: actions/setup-dotnet@v1
         with:
           dotnet-version: ${{ matrix.dotnet-version }}
-      - name: Run the app
+      - name: Run and Test the app
         run: |
           dotnet run --project DOTNET-example/Web-App &
-      - name: Sleep for 30 seconds
-        uses: jakejarvis/wait-action@master
-        with:
-          time: '30s'
-      - name: Send a sample request to localhost:5114
-        run: |
           TEMP_OUTPUT_FILE=$(mktemp)
           HTTP_CODE=$(curl --silent --output $TEMP_OUTPUT_FILE --write-out "%{http_code}" "http://localhost:5114")
           if [[ ${HTTP_CODE} -lt 200 || ${HTTP_CODE} -gt 299 ]] ; then
             >&2 cat $TEMP_OUTPUT_FILE
             exit 1;
           fi
           cat $TEMP_OUTPUT_FILE
-          rm $TEMP_OUTPUT_FILE
+          rm $TEMP_OUTPUT_FILE    
```

Comment:
Looks like some unnecessary whitespace snuck in

Reply:
I think these files are outdated.